ToDoTime/ClientApp/src/App.tsx

```diff
@@ -1,27 +1,61 @@
-import React from "react";
-import logo from "./logo.svg";
-import styles from "./App.module.scss";
-import styled from "@emotion/styled";
+import { LoaderFunctionArgs, RouterProvider, createBrowserRouter } from "react-router-dom";
+import { Layout } from "pages/Layout";
+import { Home } from "pages/home/Home";
+import {
+  homeRoute,
+  listsRoute,
+  newListRoute,
+  routes,
+  taskDetailRoute,
+  tasksListDetailsRoute
+} from "routes";
+import { NewList } from "pages/NewList";
+import { ErrorPage } from "pages/ErrorPage";
+import { TaskDetail } from "pages/TaskDetail";
+import { Lists } from "pages/Lists";
+import { TasksListDetails } from "pages/TasksListDetails";
+
+interface TasksListDetailsParams {
+  listId: string;
+}
+
+const tasksListDetailsLoader = async ({ params }: LoaderFunctionArgs<TasksListDetailsParams>) => {
```

Comment:
Loader should be in a separate file, otherwise this file will contain all logics for other pages as well.

Reply:
Added to a separated file.